src/pages/thanks-page/index.js

```diff
@@ -0,0 +1,3 @@
+import Thanks from './thanks-page';
```

Comment:
Please, rename `Thanks` to `ThanksPage` because the name of the exported component and the name of the file should match

Reply:
Done